audio/portaudio/Portfile

```diff
@@ -23,38 +25,37 @@ long_description    PortAudio is a free, cross platform, open-source, audio I/O
 homepage            http://www.portaudio.com/
 master_sites        ${homepage}archives
 
-distname            pa_stable_v[strsed ${version} {g/\./_/}]
+distname            pa_stable_v${major}[format %04d [expr {$minor * 100}]]_${reldate}
 extract.suffix      .tgz
 
-checksums           rmd160  0a290cea8a102a45b2df9b72046b3eed0ab35b4f \
-                    sha256  8fe024a5f0681e112c6979808f684c3516061cc51d3acc0b726af98fc96c8d57
+checksums           rmd160  e6e5cd3f3cb7469aa17549c189e445d573567e13 \
+                    sha256  f5a21d7dcd6ee84397446fa1fa1a0675bb2e8a4a6dceb4305a8404698d8d1513
 
 worksrcdir          ${name}
 
 # By default, PortAudio is built as a universal binary.
 # To build a non-universal library for the host architecture,
 # simply use the --disable-mac-universal option with configure.
 # http://www.portaudio.com/trac/wiki/TutorialDir/Compile/MacintoshCoreAudio
-configure.args			\
-    --enable-option-checking	\
-    --enable-cxx		\
-    --disable-mac-universal	\
-    --without-alsa		\
-    --without-jack		\
-    --without-oss		\
-    --without-asihpi		\
+configure.args      \
+    --disable-mac-universal \
+    --enable-cxx \
+    --enable-option-checking \
+    --without-alsa      \
+    --without-jack      \
+    --without-oss       \
+    --without-asihpi        \
     --without-winapi
```

Comment:
Does this just rearrange the ordering of these?

Reply:
Yeah.

Reply:
I wouldn't, as it makes the review harder, but if you're going to, remove the tabs.